orchestra/communication/slack.py

```diff
@@ -1,28 +1,40 @@
 import random
 import string
 
+import slacker
 from django.conf import settings
 from django.utils.text import slugify
-from slacker import Slacker
 
 from orchestra.utils.decorators import run_if
 from orchestra.communication.errors import SlackFormatError
 
+import logging
+logger = logging.getLogger(__name__)
+
 # Types of responses we can send to slack
 VALID_RESPONSE_TYPES = {'ephemeral', 'in_channel'}
+_request = slacker.BaseAPI._request
+
+
+def _silent_request(*args, **kwargs):
```

Comment:
docstring explaining this and a TODO saying we might want to surface certain errors